README.md

```diff
@@ -58,19 +58,22 @@ Once the all dependencies have been installed, start the Flask app in developmen
 $ poetry run flask run
 ```
 
-You should see output similar to the following:
+The containerised development version of the app (with hot reloading) can be run with the following docker commands:
+
 ```bash
- * Serving Flask app 'todo_app/app'
- * Debug mode: on
-WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
- * Running on http://127.0.0.1:5000
-Press CTRL+C to quit
- * Restarting with stat
- * Debugger is active!
- * Debugger PIN: 113-666-066
+$ docker build --target dev --tag todo-app:dev .
+$ docker run --rm -it --env-file .env -p 5000:5000 -v ./todo_app/:/opt/app/todo_app/ todo-app:dev
```

Comment:
Docker doesn't support relative paths directly (at least, unless it's different on Mac/a later update?)😞 Compose will happily handle this for you, but if we want a cross-platform command here we'll need to fudge that ourselves:
```suggestion
$ docker run --rm -it --env-file .env -p 5000:5000 -v "${pwd}/todo_app/:/opt/app/todo_app/" todo-app:dev
```

Reply:
I've used relative paths whenever I've used this 🤷 

I am in WSL with docker version 26.0.0